src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -102,3 +109,50 @@ def get_projects_and_filenames_from_source_file(
                 pr_filename.replace(code_mapping.source_root, code_mapping.stack_root)
             )
     return project_list, sentry_filenames
+
+
+def get_top_5_issues_by_count_for_file(
+    projects: List[Project], sentry_filenames: List[str]
+) -> list[dict[str, Any]]:
+    """Given a list of issue group ids, return a sublist of the top 5 ordered by event count"""
+    group_ids = list(
+        Group.objects.filter(
+            last_seen__gte=datetime.now() - timedelta(days=14),
+            project__in=projects,
+        ).values_list("id", flat=True)
+    )
+    project_ids = [p.id for p in projects]
+
+    request = SnubaRequest(
+        dataset=Dataset.Events.value,
+        app_id="default",
+        tenant_ids={"organization_id": projects[0].organization_id},
```

Comment:
Nice stuff! One thing I would suggest is if we could put this behind a new referrer so we could monitor its impact and allocate the appropriate resources to this type of query accordingly.

Reply:
it's here https://github.com/getsentry/sentry/blob/e1d5976683bd7534910abf88efee67bfd0be27f2/src/sentry/tasks/integrations/github/open_pr_comment.py#L158

Reply:
Great and it gets set in the tenant_id in `raw_snql_query()`. Thank you.